.github/workflows/land-blocking.yml

```diff
@@ -71,12 +71,23 @@ jobs:
         run: |
           export CTI_OUTPUT_LOG=$(mktemp)
           echo "::set-env name=CTI_OUTPUT_LOG::$CTI_OUTPUT_LOG"
-          JUMPHOST=${{secrets.CLUSTER_TEST_JUMPHOST}} \
+          if [[ ${{ secrets.ACTIONS_LBT_USE_K8S }} == true ]]; then
```

Comment:
Might as well make k8s default.

Reply:
Since you set `ACTIONS_LBT_USE_K8S` to `true`, I think this should be fine for now